src/release-verification.js

```diff
@@ -75,7 +67,8 @@ module.exports = robot => {
         'release-pr-head-sha': pr.head.sha,
         'release-pr-head-repo-full-name': pr.head.repo.full_name,
         'release-pr-base-repo-full-name': pr.base.repo.full_name,
-        'release-pr-prerelease': String(prerelease),
+        'release-pr-prerelease': String(isPrerelease),
```

Comment:
Do we use this anywhere?  A [quick search](https://github.com/search?q=release-pr-prerelease+org%3Afusionjs+org%3Auber-web+org%3Auber-workflow&type=Code) doesn't turn up anything.